snapshot-labs/sx-monorepo
# feat: unify settings into single EditSettings page

### Summary

This PR merges Settings and EditSettings into single page.
It also adds reusable `ModalTransactionProgress` modal that you can use to track actions (can be also used for Voting progress).

Closes: https://github.com/snapshot-labs/workflow/issues/165

[Designs on Figma](https://www.figma.com/design/hmHTau1Y1KYzrBBKLQr5QV/%23design?node-id=5248-3827&t=IGnSKZSczvUiHerC-0)

### How to test

1. Go to space you are controller of (onchain only).
2. You see Settings tab and you can open it.
3. You can see different settings (strategies, voting, executions, controller).
4. You can modify those settings and changes are detected (there is some delay because metadata needs to be created async).
5. You can reset changes.
6. You can modify settings and go through new ModalTransactionProgress flow (you can reject in the middle of it, wait for confirmation etc.
7. You can modify controller and go through new ModalTransactionProgress flow.

### Screenshots


https://github.com/user-attachments/assets/5aac8fd8-aeb6-43a7-a925-443b86b685d2
